libs/ui/inputs/src/lib/dropdown-multiselect/dropdown-multiselect.component.html

```diff
@@ -8,15 +8,22 @@
   cdkOverlayOrigin
   #overlayOrigin="cdkOverlayOrigin"
 >
-  <div class="grow text-left truncate font-medium mr-2">
-    {{ title }}
+  <div class="grow flex items-center mr-2 gap-2">
+    <div class="text-left truncate font-medium">{{ title }}</div>
+    <div
+      *ngIf="hasSelectedChoices"
+      class="shrink-0 rounded-full bg-primary text-white font-bold text-[12px] w-5 h-5 flex items-center justify-center mr-1 selected-count"
+    >
+      {{ selected.length }}
+    </div>
   </div>
-  <div
-    *ngIf="hasSelectedChoices"
-    class="shrink-0 rounded-full bg-primary text-white font-bold text-[13px] w-6 h-6 p-[6px] mr-1 selected-count"
+  <mat-icon
```

Comment:
This won't be accessible, it should be a `<button>`

Reply:
I've added a button tag around it